src/Form/FieldSelect.js

```diff
@@ -0,0 +1,153 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import Dropdown from '../Dropdown/Dropdown';
+
+class FieldSelect extends React.Component {
+
+  handleChange(selectedValue) {
+    this.props.handleEvent('change',
+      this.props.name, selectedValue);
+  }
+
+  hasError() {
+    let {props} = this;
+    let validationError = props.validationError;
+    return !!(validationError && validationError[props.name]);
+  }
+
+  getDropDown(dropdownItems) {
+    return (
+      <Dropdown buttonClassName="button dropdown-toggle"
+        dropdownMenuClassName="dropdown-menu"
+        dropdownMenuListClassName="dropdown-menu-list"
+        items={dropdownItems}
+        onItemSelection={this.handleChange.bind(this)}
+        transition={true}
+        wrapperClassName="dropdown"/>
+    );
+  }
+
+  getErrorMsg() {
+    let errorMsg = null;
+    let {props} = this;
+
+    if (this.hasError()) {
+      errorMsg = (
+        <p className={props.helpBlockClass}>
+          {props.validationError[props.name]}
+        </p>
+      );
+    }
+
+    return errorMsg;
+  }
+
+  getLabel() {
+    let {label, showLabel} = this.props;
+
+    if (!showLabel) {
+      return null;
+    }
+
+    if (typeof showLabel === 'string') {
+      label = showLabel;
+    }
+
+    if (typeof showLabel !== 'string' && showLabel !== true) {
+      return showLabel;
+    }
+
+    return (
+      <label>{label}</label>
+    );
+  }
+
+  render() {
+    let {
+      columnWidth, formGroupClass, formElementClass, formGroupErrorClass,
+      options
+    } = this.props;
+
+    options = this.props.options.map(function (option) {
```

Comment:
We can just use `options` again instead of `this.props.options`, so `options = options.map(...`